CHANGELOG.rst

```diff
@@ -1,11 +1,21 @@
 Changelog
 =========
 
+0.4.0
+-----
+
+* Added methods for create_building, update_building, get_columns, create_extra_data_column, create_extra_data_column_from_file, retrieve_at_building_and_update, retrieve_portfolio_manager_property, and import_portfolio_manager_property in https://github.com/SEED-platform/py-seed/pull/22
```

Comment:
The documentation is lacking, we are now pulling this directly from github. I'll update...